contracts/MerkleFactory.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/proxy/Clones.sol";
+import "./MerkleDrop.sol";
+
+contract MerkleFactory {
+    MerkleDrop[] public drops;
```

Comment:
What will be the max size of drops this contract can store? what if Snapshot has 100K drops created?

Reply:
There are no limits in the specification, so arrays may grow up to 2^256-1 elements. Similar to other [comment](https://github.com/snapshot-labs/drop/pull/1#discussion_r777107945), we are doing it as clones so 100k drops will have 100k clone contracts. Each has its state and size.